src/com/estafet/learning/US005/Task5.java

```diff
@@ -0,0 +1,142 @@
+package com.estafet.learning.US005;
+
+import java.util.Scanner;
+
+public class Task5 {
+    public static void main(String[] args) {
+        checkNumberIsEvenOrOdd();
+        fibonacii();
+        factorial();
+        factorialOfOddNumbers();
+        addTwoNumbers();
+        numberOfDaysInMonth();
+    }
+
+    public static int number;
+
+    public static void checkNumberIsEvenOrOdd() {
+        Scanner reader = new Scanner(System.in);
+
+        System.out.format("Enter a number to check is it even or odd: %n");
+        int num = reader.nextInt();
+
+        if (num % 2 == 0)
+            System.out.format(num + " is even%n");
+        else
+            System.out.format(num + " is odd%n");
+    }
+
+    public static void fibonacii() {
+        Scanner reader = new Scanner(System.in);
+        System.out.format("Enter a number to view Fibonacci series: %n");
+
+        do {
+            System.out.format("Please enter a positive number!%n");
+            while (!reader.hasNextInt()) {
+                System.out.format("That's not a number!%n");
+                System.out.format("Enter a number to view Fibonacci series: %n");
+                reader.next();
+            }
+            number = reader.nextInt();
+        } while (number <= 0);
+
+        int n1 = 0, n2 = 1, n3, i;
```

Comment:
variable `i` is not needed in the local scope, can be declared in loop

Reply:
ok, changed.